### PR TITLE
Unit test commandlinehelper

### PR DESCRIPTION
**Pull request by denipermana01**
Added unit test for command line helper.

### CODE REVIEW

**Comment by denipermana01** [comment]
@ratnadeep-olx @emanueldelgado please review

File: test/CommandLineHelperTest.php [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+ * User: denipermana
+ * Date: 5/4/16
+ * Time: 4:59 PM
+ */
+class CommandLineHelperTest extends \PHPUnit_Framework_TestCase
+{
+    /** @var CommandLineHelperTestFunction */
+    private $commandLine;
+
+    private $container;
```
Comment (newcron) [inline test/CommandLineHelperTest.php]: Codacy Issue found: [Avoid unused private fields such as '$container'.](https://www.codacy.com/app/Atlas-Development-Team/atlas-web-framework-orm/file/2520747514/issues/source?bid=3255648&fileBranchId=3280313#l23)

Comment (newcron) [inline test/CommandLineHelperTest.php]: Codacy Issue found: [Avoid unused private fields such as '$container'.](https://www.codacy.com/app/Atlas-Development-Team/atlas-web-framework-orm/file/2520786117/issues/source?bid=3255648&fileBranchId=3280313#l23)

File: test/AnnotationDriverFactoryTest.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<?php
+namespace naspersclassifieds\shared\atlas\web\framework\orm\test;
+
+use naspersclassifieds\shared\atlas\web\framework\orm\AnnotationDriverFactory;
+use naspersclassifieds\shared\atlas\web\framework\orm\AtlasOrmAnnotationDriver;
+
+class AnnotationDriverFactoryTest extends \PHPUnit_Framework_TestCase
+{
+    /* @var AnnotationDriverFactory */
+    private $annotationDriver;
```
Comment (newcron) [inline test/AnnotationDriverFactoryTest.php]: Codacy Issue found: [Avoid unused private fields such as '$annotationDriver'.](https://www.codacy.com/app/Atlas-Development-Team/atlas-web-framework-orm/file/2520988031/issues/source?bid=3255648&fileBranchId=3280313#l10)

**Pull request merged**